docs/INSTALL-devel.rst

```diff
@@ -0,0 +1,290 @@
+Following guide covers setting up a developement environment for HaaS
```

Comment:
I would avoid rst unless it's specifically for read the docs. md is just easier and more fun.

BTW - pandoc can convert things for you.